processor/websocketprocessor/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/websocketprocessor
+
+go 1.19
+
+require (
+	github.com/stretchr/testify v1.8.2
+	go.opentelemetry.io/collector v0.75.0
+	go.opentelemetry.io/collector/component v0.75.0
```

Comment:
You are going to need to update versions here.

Reply:
Thanks. Should be done now.